### DISCUSSION TITLE
Add the possibility to search the internet

### DISCUSSION THREAD

**Discussion by RocketKevQt** [open-webui/open-webui#1519] [General]
I like what phind is doing. They're sythetizing what you're asking then searching through the web adding part of the result into the prompt and then giving you an answer, like what WebChatGpt is also doing. Would be nice to have the possiblity to search through the web when asking something

a solution like WebChatGpt, maybe using startpage to anonymize the results

The alternative would be to use a web extension to do that but there is none for this interface sadly

this could fix partly the problem with local coding assistant that needs to be often updated because of technical debt

**Comment by spidercraft** [comment]
that would be really cool ngl, the current internet integration is alright, but it would be good if models could search automatically too as a option kinda like webChatGpt

**Comment by arsaboo** [comment]
See https://github.com/open-webui/open-webui/issues/586
